store/user.ts

```diff
@@ -33,7 +33,7 @@ export const mutations: MutationTree<UserState> = {
   },
   CLEAR_USER(state: UserState) {
     state.serverUrl = '';
-    state.serverUrl = '';
+    state.serverUrl = 'http://127.0.0.1:8096';
```

Comment:
Same as above, why?

Reply:
Somewhat sane default for the state of the store. Not necessary, but it'll be populated with the contents from the JSON config further down the line. Currently, it's pretty much overridden as soon as you click on "Login" or when the client boots.

Reply:
@mrtimscampi If it's not strictly necessary I would rather have it blank instead. In case it is necessary, document it.

We don't want to have arbitrary things in our new stuff like we do right now in our current web and server :)